### ISSUE TITLE
Fail to acquire debugger for single-Dockerfile debugging after compose up debugging

### ISSUE THREAD

**Issue by bwateratmsft** [microsoft/vscode-docker#1743]
**Repro steps:**

1. Completely remove any existing `~/.vsdbg` folder (to simulate first time run)
1. Create a .NET Core project and add Docker files, including compose
1. Compose up docker-compose**.debug**.yml. This maps `~/.vsdbg` to `/remote_debugger`.
1. Run the "Docker .NET Core Attach (Preview)" launch config. It will prompt to install debugger, allow it to do so.
1. Compose down. The debugger remains installed on the _host_, at `~/.vsdbg`, because it was volume mapped and then installed in the container.
1. Switch to "Docker .NET Core Launch" and try to F5.

Note, I was using an Alpine image when I hit this, but that might not matter.

This fails because when the debugger was installed to `~/.vsdbg`, it has `root` as owner and group, and the local user has no permissions:

```
Acquiring the latest .NET Core debugger...
Unable to acquire the .NET Core debugger.
Error: EACCES: permission denied, open '/home/brandon/.vsdbg/getvsdbg.sh'
```

**Workaround:**

1. `chown` the `~/.vsdbg` directory to the current user
1. Completely remove the `~/.vsdbg` directory

IMO, this is a minor bug with an easy workaround but it isn't obvious to users what happened. I wanted to seed this issue in case people stumble upon it, but I'm OK with won't-fixing this unless we get explicit user feedback.

**Comment by bwateratmsft** [comment]
We talked about it; probably the simplest solution is to inspect the container, if `/remote_debugger` is a mapped volume we will not install vsdbg. If it's also not present, we should show a clear error.

**Comment by bwateratmsft** [comment]
This will be fixed by the fix for #1794
